CONTRIBUTORS.txt

```diff
@@ -91,16 +103,20 @@ needs to be listed here.
 - Markus Kieselmann <markus@kieselmann.eu>
 - Marli Oshlack <marli.oshlack@apple.com>
 - Matt Eaton <agnosticdev@gmail.com>
+- Matteo Comisso <matteo.comisso@me.com>
 - Max Desiatov <max@desiatov.com>
 - Max Moiseev <moiseev@apple.com>
+- Maxim Zaks <maxim.zaks@gmail.com>
 - Moritz Lang <hi@slashmo.codes>
 - Nathan Harris <nathan@mordil.info>
 - Nethra Ravindran <nethra.ravindran@in.ibm.com>
 - Norio Nomura <norio.nomura@gmail.com>
 - Norman Maurer <norman_maurer@apple.com>
 - Ole Begemann <ole@oleb.net>
 - OleG <emptystamp@gmail.com>
+- Paul Schmiedmayer <PSchmiedmayer@users.noreply.github.com>
```

Comment:
@PSchmiedmayer  if you would like to have a real email address here, please add a comment with your preferred address. Thank you!

Reply:
@weissi Thanks for updating the contributors list! You can use paul.schmiedmayer@tum.de as a real email address that I use.